Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
```

Comment:
Don't declare all variables at the same place. YOu should declare them right next to where you plan to use them (next to parse. It's not good, because you will have to scroll from declaration back to consumptions and there is simply no point to put the responsibility of remembered about them across a whole function scope.